openadapt/record.py

```diff
@@ -197,23 +206,53 @@ def process_events(
                 num_video_events.value += 1
         elif event.type == "window":
             prev_window_event = event
+        elif event.type == "browser":
+            # logger.info("Processing browser event")
+            prev_browser_event = event
+            # process_event(
+            #     prev_browser_event,
+            #     browser_write_q,
+            #     write_browser_event,
+            #     recording,
+            #     perf_q,
+            # )
+            # num_browser_events.value += 1
```

Comment:
Should the commented out lines here be removed?

Reply:
Done here: 6a3b74b4f318a41e73824cc344a1f578d36732b4